app/Http/Controllers/Order/OrderController.php

```diff
@@ -193,14 +194,22 @@ public function show($id)
         try {
             $order = $this->order->findOrFail($id);
             $subscription = $order->subscription()->first();
-            $currenctVersion = $subscription->version;
-            $lastActivity = $subscription->updated_at;
-            $invoiceid = $order->invoice_id;
-            $invoice = $this->invoice->where('id', $invoiceid)->first();
+            $date = '--';
+            $licdate = '--';
+            $supdate = '--';
+            if($subscription) {
+              $date =  strtotime($subscription->update_ends_at)>1 ? $this->getExpiryLabel($subscription->update_ends_at): '--';
+              $licdate = strtotime($subscription->ends_at)>1 ? $this->getExpiryLabel($subscription->ends_at) :'--' ;
+              $supdate =  strtotime($subscription->support_ends_at)>1 ? $this->getExpiryLabel($subscription->support_ends_at):'--' ;
```

Comment:
why `>1`?

Reply:
When timestamp value is 0000-00-00 00:00:00 in db it returns negative value